<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiShortcutDialog</class>
 <widget class="QDialog" name="GuiShortcutDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QCheckBox" name="mp_cbUseShortcuts">
     <property name="text">
      <string>Use custom shortcuts</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTreeWidget" name="mp_twShortcuts">
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="mp_pbDefault">
     <property name="statusTip">
      <string>Restore default shortcuts</string>
     </property>
     <property name="text">
      <string>Restore</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>136</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="mp_pbOk">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="mp_pbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_cbUseShortcuts</tabstop>
  <tabstop>mp_twShortcuts</tabstop>
  <tabstop>mp_pbDefault</tabstop>
  <tabstop>mp_pbOk</tabstop>
  <tabstop>mp_pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
